gnome-initial-setup/gis-driver.c

```diff
@@ -196,6 +237,19 @@ gis_driver_get_personality (GisDriver *driver)
   return priv->personality;
 }
 
+gboolean
+gis_driver_is_small_screen (GisDriver *driver)
```

Comment:
You don't need the driver here. I wonder if this shouldn't just be an utility function.